src/stactools/sentinel2/stac.py

```diff
@@ -121,15 +127,25 @@ def create_item(
         metadata = metadata_from_granule_metadata(
             granule_href, read_href_modifier, tolerance
         )
+    created = datetime.now().strftime("%Y-%m-%dT%H:%MZ")
 
     item = pystac.Item(
         id=metadata.scene_id,
         geometry=metadata.geometry,
         bbox=metadata.bbox,
         datetime=metadata.datetime,
-        properties={},
+        properties={"created": created},
     )
 
+    # Handle antimeridian
+    handle_am = False
+    for coord in metadata.geometry["coordinates"][0]:
+        if coord[0] >= 180 or coord[0] <= -180:
+            handle_am = True
```

Comment:
Just clarifying, when you have a scene that crosses the antimeridian, there will be one or more points in the geometry will have a longitude value less than -180 or greater than +180? I'm accustomed to seeing a mix of positive and negative longitude values, but none with a magnitude greater than 180.